maths/area.py

```diff
@@ -5,7 +5,7 @@
 from typing import Union

```

Comment:
Travis not happy... ./maths/area.py:5:1: F401 'typing.Union' imported but unused

Reply:
I swear that line keeps coming back by itself :)